lib/WorkflowMag.groovy

```diff
@@ -83,35 +85,36 @@ class WorkflowMag {
             }
         }
 
-        // Check if at least two binners were applied in order to run DAS Tool for bin refinment
-        // (needs to be adjusted in case additional binners are added)
-        if (params.refine_bins_dastool && params.skip_metabat2 ) {
-            log.error 'Both --refine_bins_dastool and --skip_metabat2 are specified! Invalid combination, bin refinement requires MetaBAT2 and MaxBin2 binning results.'
-            System.exit(1)
+        // Check MetaBAT2 inputs
+        if ( !params.skip_metabat2 && params.min_contig_size < 1500 ) {
+            log.warn "Specified min. contig size under minimum for MetaBAT2. MetaBAT2 will be run with 1500 (other binners not affected). You supplied: --min_contig_size ${params.min_contig_size}"
         }
-        if (params.refine_bins_dastool && params.skip_maxbin2 ) {
-            log.error 'Both --refine_bins_dastool and --skip_maxbin2 are specified! Invalid combination, bin refinement requires MetaBAT2 and MaxBin2 binning results.'
+
+        // Check more than one binner is run for bin refinement  (required DAS by Tool)
+        // If the number of run binners (i.e., number of not-skipped) is more than one, otherwise throw an error
+        if ( params.refine_bins_dastool && !([ params.skip_metabat2, params.skip_maxbin2, params.skip_concoct ].count(false) > 1) ) {
+            log.error 'Bin refinement with --refine_bins_dastool requires at least two binners to be running (not skipped). Check input.'
             System.exit(1)
         }
 
         // Check that bin refinement is actually turned on if any of the refined bins are requested for downstream
         if (!params.refine_bins_dastool && params.postbinning_input != 'raw_bins_only') {
-            log.error 'The parameter '--postbinning_input ${params.postbinning_input}' for downstream steps can only be specified if bin refinement is activated with --refine_bins_dastool! Check input.'
+            log.error 'The parameter '--postbinning_input ${ params.postbinning_input }' for downstream steps can only be specified if bin refinement is activated with --refine_bins_dastool! Check input.'
             System.exit(1)
         }
 
         // Check if BUSCO parameters combinations are valid
-        if (params.skip_busco) {
+        if (params.skip_binqc && params.binqc_tool == 'busco') {
```

Comment:
```suggestion
        if (params.skip_binqc && params.binqc_tool == 'checkm') {
            log.error 'Both --skip_binqc and --binqc_tool \'checkm\' are specified! Invalid combination, please specify either --skip_binqc or --binqc_tool.'
            System.exit(1)
        }
        if (params.skip_binqc) {
```

If someone specifies CheckM and --skip_binqc, this should maybe better cause an error.